WordPress/Docs/DB/PreparedSQLStandard.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Prepared SQL"
+    >
+    <standard>
+    <![CDATA[
+    When making direct database queries using $wpdb, you should use $wpdb->prepare(), and use placeholders for all variables used in the query. You should not use variable interpolation or concatenation.
```

Comment:
I wonder if something along the lines of the below would make the description clearer ?

```suggestion
    When querying the database, you should use $wpdb->prepare() to escape and quote the contents of variables. This prevents SQL injection.
    Use placeholders for all variables used in the query. You should not use variable interpolation or concatenation.
```